### DISCUSSION TITLE
Failed to connect to relay server: Please try later

### DISCUSSION THREAD

**Discussion by Wa111ng** [rustdesk/rustdesk-server-pro#436] [Q&A]
Only able to connect if both PCs are on same network. On a different network even though client shows "Ready", not able to connect successfully Error: "Failed to connect to relay server: Please try later"

**Comment by rustdesk** [comment]
`hbbr` is the relay server. `ready` does only indicate `hbbs` works.

https://rustdesk.com/docs/en/self-host/#how-does-self-hosted-server-work
